# Translations template for vigilo-vigiconf-local.
# Copyright (C) 2017 ORGANIZATION
# This file is distributed under the same license as the vigilo-vigiconf-local
# project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2017.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: vigilo-vigiconf-local 4.1.0rc2\n"
"Report-Msgid-Bugs-To: support.vigilo@csgroup.eu\n"
"POT-Creation-Date: 2017-08-24 12:17+0200\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: support.vigilo@csgroup.eu\n"
"Language-Team: LANGUAGE <support.vigilo@csgroup.eu>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 0.9.4\n"

#: src/vigilo/vigiconf_local/__init__.py:33
msgid "%prog [options] command [arguments]"
msgstr ""

#: src/vigilo/vigiconf_local/__init__.py:35
msgid "Only print the command that would be run"
msgstr ""

#: src/vigilo/vigiconf_local/__init__.py:38
#, python-format
msgid "No command selected. Available commands: %s"
msgstr ""

#: src/vigilo/vigiconf_local/__init__.py:43
#, python-format
msgid "Unknown command. Available commands: %s"
msgstr ""

#: src/vigilo/vigiconf_local/__init__.py:59
#, python-format
msgid "Wrong number of arguments. Expected: %s"
msgstr ""

#: src/vigilo/vigiconf_local/__init__.py:70
#, python-format
msgid "Setup error: %s"
msgstr ""

#: src/vigilo/vigiconf_local/__init__.py:73
#, python-format
msgid "Error: %s"
msgstr ""

#: src/vigilo/vigiconf_local/commands.py:129
#, python-format
msgid "The archive '%s' does not exist, copy it first"
msgstr ""

#: src/vigilo/vigiconf_local/commands.py:148
#, python-format
msgid "Can't untar the configuration: %(output)s"
msgstr ""

#: src/vigilo/vigiconf_local/commands.py:190
msgid "Please specify an application name to validate"
msgstr ""

#: src/vigilo/vigiconf_local/commands.py:194
#, python-format
msgid "No validation script: %s"
msgstr ""

#: src/vigilo/vigiconf_local/commands.py:212
#, python-format
msgid "Validation failed for application '%(app)s'. Output: %(output)s"
msgstr ""

#: src/vigilo/vigiconf_local/commands.py:234
msgid "The 'new' directory does not exist. Deploy the configuration first."
msgstr ""

#: src/vigilo/vigiconf_local/commands.py:241
msgid "Backing up the directory 'prod' to 'old', and renaming 'new' into 'prod'"
msgstr ""

#: src/vigilo/vigiconf_local/commands.py:257
#, python-format
msgid ""
"Configuration activation failed: %(error)s. User '%(user)s' must have write "
"access to '%(dir)s'."
msgstr ""

#: src/vigilo/vigiconf_local/commands.py:265
#, python-format
msgid "Configuration activation failed: %s."
msgstr ""

#: src/vigilo/vigiconf_local/commands.py:290
#, python-format
msgid "The script '%(script)s' does not exist for application '%(app)s'"
msgstr ""

#: src/vigilo/vigiconf_local/commands.py:310
#, python-format
msgid ""
"Action %(action)s failed for application %(app)s. Code: %(code)d. Output: "
"%(output)s"
msgstr ""

#: src/vigilo/vigiconf_local/commands.py:357
#, python-format
msgid "The directory '%s' does not exist."
msgstr ""

#: src/vigilo/vigiconf_local/commands.py:363
#, python-format
msgid "Getting revisions from these directories: %s"
msgstr ""

#: src/vigilo/vigiconf_local/commands.py:397
msgid "The directory 'new' does not exist."
msgstr ""

#: src/vigilo/vigiconf_local/commands.py:403
#, python-format
msgid "Setting revision to: %s"
msgstr ""

